index.html

```diff
@@ -1,19 +1,74 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
     <title>Karma</title>
-    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,300" rel="stylesheet" type="text/css" />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
 </head>
-<body>
 
+<body>
     <!-- Add your HTML markup here -->
+    <header>
+        <img src="img/karma-logo.svg" alt="Logo of the website" />
+        <nav>
+            
+            <ul>
+                <li><a href="#">Meet Karma</a></li>
+                <li><a href="#">How it Works</a></li>
+                <li><a href="#">Store</a></li>
+                <li><a href="#">Blog</a></li>
+                <li><a href="#">Help</a></li>
+                <li><a href="#">Login</a></li>
+            </ul>
+            <a class="burger-nav" id="burger-btn"></a>
+
+        </nav>
+    </header>
+    <main>
+        <!-- Background will be added in css -->
+        <section class="hero">
+            <h1>Introducing Karma</h1>
+            <h2>Bring WiFi with you, everywhere you go.</h2>
+            <button>Learn More</button>
+        </section>
+        <section class="features">
+            <h2>Everyone needs a little Karma.</h2>
+            <section class="icon-container">
+                <section class="icons">
+                    <img src="img/icon-devices.svg" alt="Image 
+                of different internet devices" />
+                    <p>Internet for all devices</p>
+                </section>
+                <section class="icons">
+                    <img src="img/icon-coffee.svg" alt="Image of a coffee cup" />
+                    <p>Boost your productivity</p>
+                </section>
+                <section class="icons">
+                    <img src="img/icon-refill.svg" alt="Gas refill tank image" />
```

Comment:
```suggestion
                    <img src="img/icon-refill.svg" alt="Gas refill tank" />
```
Same as my other comment